docs/quickstarts/chrome-extension.mdx

```diff
@@ -0,0 +1,252 @@
+
+  import '~style.css'
+
+  const PUBLISHABLE_KEY = process.env.PLASMO_PUBLIC_CLERK_PUBLISHABLE_KEY
+
+  if (!PUBLISHABLE_KEY) {
+    throw new Error('Please add the PLASMO_PUBLIC_CLERK_PUBLISHABLE_KEY to the .env.development file')
+  }
+
+  function IndexPopup() {
+    return (
+      <ClerkProvider publishableKey={PUBLISHABLE_KEY}>
+        <div className="plasmo-flex plasmo-items-center plasmo-justify-center plasmo-h-16 plasmo-w-40">
+          <CountButton />
+        </div>
+      </ClerkProvider>
+    )
+  }
+  ```
+
+  ## Create a header with Clerk components
+
+  You can control which content signed in and signed out users can see with Clerk's [prebuilt components](/docs/components/overview). To get started, create a header for your users to sign in or out. With Chrome Extensions, you can also add this logic to a footer. To do this, you will use:
+
+  - [`<SignedIn>`](/docs/components/control/signed-in): Children of this component can only be seen while **signed in**.
+  - [`<SignedOut>`](/docs/components/control/signed-out): Children of this component can only be seen while **signed out**.
+  - [`<UserButton />`](/docs/components/user/user-button): A prebuilt component that comes styled out of the box to show the avatar from the account the user is signed in with.
+  - [`<SignInButton />`](/docs/components/unstyled/sign-in-button): An unstyled component that links to the sign-in page or displays the sign-in modal.
+
+  ```jsx {{ filename: 'src/popup.tsx', mark: [1, [15, 24], 26] }}
+  import {
+    ClerkProvider,
+    SignInButton,
+    SignedIn,
+    SignedOut,
+    UserButton,
+  } from '@clerk/chrome-extension'
+  import { CountButton } from '~features/count-button'
+
+  import '~style.css'
+
+  const PUBLISHABLE_KEY = process.env.PLASMO_PUBLIC_CLERK_PUBLISHABLE_KEY
+
+  if (!PUBLISHABLE_KEY) {
+    throw new Error('Please add the PLASMO_PUBLIC_CLERK_PUBLISHABLE_KEY to the .env.development file')
+  }
+
+  function IndexPopup() {
+    return (
+      <ClerkProvider publishableKey={PUBLISHABLE_KEY}>
+        <div className="plasmo-flex plasmo-items-center plasmo-justify-center plasmo-h-[600px] plasmo-w-[800px] plasmo-flex plasmo-flex-col">
+          <header className="plasmo-w-full">
+            <SignedOut>
+              <SignInButton mode="modal" />
+            </SignedOut>
+            <SignedIn>
+              <UserButton />
+            </SignedIn>
+          </header>
+          <main className="plasmo-grow">
+            <CountButton />
+          </main>
+        </div>
+      </ClerkProvider>
+    )
+  }
+
+  export default IndexPopup
+  ```
+
+  ## Create a consistent CRX ID for your extension
+
+  Chrome Extensions have a unique CRX ID that rotates by default, which can cause errors and issues with the Clerk integration. To avoid these problems, ensure that you have a **consistent** CRX ID in both development and production for your extension by following these steps:
```

Comment:
Is this correct? It rotates by default.

Reply:
Yes -- that's why we create the consistent one. Otherwise you end having it change, not realizing it and Clerk stops working.

Reply:
okay awesome - just wanted to make sure I wrote accurate info!